app/views/devise/sessions/splash.html.erb

```diff
@@ -0,0 +1,8 @@
+
+<div class="splash">
+     <div class = "main-heading"> PIERCEL PAYMENTS </div>
+    <div class='d-flex flex-column  p-3 gap-2'>
+      <%= link_to 'Sign In', new_user_session_path, class: 'btn button-login border border-light text-light  bg-primary col-3  p-2' %>
+      <%= link_to 'Sign Up', new_user_registration_path, class: 'btn button-register border border-light bg-primary text-light col-3 p-2' %>
+    </div>
+</div>
```

Comment:
- [ ] The first reviewer asked that you work on the styling of the splash page for both the development and the production stages. It is required that you design the splash page to look identical to that of the design template. Having good designs give a good impression to recruiters and other developers. It shows that you pay attention to details.  Kindly work on the styling of the splash page in both the development and production stage. 😎 💻 


| Your Design - Development  | Your Design - Production | Design Template |
| ------------------------------- | ---------------------------- | -------------------|
| ![image](https://user-images.githubusercontent.com/77106962/235501168-a3ad59e8-6f2c-49d7-8f37-da923d5cdf67.png)  | ![image](https://user-images.githubusercontent.com/77106962/235501926-2ea99dff-00ff-477a-9435-b188ce4fddf5.png) | ![image](https://user-images.githubusercontent.com/77106962/235502222-9c90ab59-86c6-429a-8ccf-bdd2573f08f3.png) |